Deprecated playbooks replacements (#28950)
demisto/content: Packs/Rapid7_Nexpose/ReleaseNotes/1_2_16.md

#### Playbooks

##### Vulnerability Handling - Nexpose

> The sub-playbooks "CVE Enrichment - Generic" and "Calculate Severity - Generic" is no longer in use and has been replaced by the sub-playbooks "CVE Enrichment - Generic v2" and "Calculate Severity - 3rd-party integrations".

Reviewer: ```suggestion
The sub-playbooks "CVE Enrichment - Generic" and "Calculate Severity - Generic" are  no longer in use and have been replaced by the sub-playbooks "CVE Enrichment - Generic v2" and "Calculate Severity - 3rd-party integrations".
```
